### PR TITLE
CRM457-1768: Caseworker remove turboframes

### PR DESCRIPTION
**Pull request by dwhenry**
## Description of change

Look at removing turboframes to simplify the back button issue

[Link to relevant ticket](https://dsdmoj.atlassian.net/browse/CRM457-1768)

### CODE REVIEW

**Comment by elken** [comment]
Trying to look at this on dev and there's no claims? I thought this shared the same store as normal dev?

**Comment by dwhenry** [comment]
@elken it does... but it only loads clains created after it was deployed....

if you log into the submit backend and run `TestData::NsmBuilder.new.build(min: 220, max: 220)` it will create a application with 220 work items abnd 110 disbursements

shout and I can walk you through it.. :)

**Comment by elken** [comment]
> @elken it does... but it only loads clains created after it was deployed....
> 
> if you log into the submit backend and run `TestData::NsmBuilder.new.build(min: 220, max: 220)` it will create a application with 220 work items abnd 110 disbursements
> 
> shout and I can walk you through it.. :)

Can look at it tomorrow then sure 😄

File: config/routes.rb [hunk 1]
```diff
@@ -54,8 +54,6 @@
       end
 
       resource :claim_details, only: [:show]
-      resource :review_and_adjusts, only: [:show]
```
Comment (elken) [inline config/routes.rb:57]: Was this a mistake? Disagrees with https://github.com/ministryofjustice/nsm-e2e-test/pull/112

Comment (dwhenry) [reply config/routes.rb:57]: so changed it twice so will need a new e2e PR... :)

**Review by patrick-laa — APPROVED** [review]

**Pull request merged**